spec/features/categories/02_user_visits_category_page_spec.rb

```diff
@@ -1,15 +1,15 @@
 require "rails_helper"
 
 feature "user visits meme category page" do
-  scenario "user visits category page" do
-    evil_kermit = Category.create(name:"Evil Kermit", image_url: "kermit.com")
-    coffee_kermit = Category.create(name: "Coffee Kermit", image_url: "kermit.com")
-
-    visit category_path(evil_kermit)
-
-    expect(page).to have_content "Evil Kermit"
-    expect(page).to have_xpath("//img[contains(@src,'kermit.com')]")
-  end
+  # scenario "user visits category page" do
+  #   evil_kermit = Category.create(name:"Evil Kermit", image_url: "kermit.com")
+  #   coffee_kermit = Category.create(name: "Coffee Kermit", image_url: "kermit.com")
+  #
+  #   visit category_path(evil_kermit)
+  #
+  #   expect(page).to have_content "Evil Kermit"
+  #   expect(page).to have_xpath("//img[contains(@src,'kermit.com')]")
+  # end
```

Comment:
I am not a huge fan of merging commented out code into master (especially when it already exists in the git history.) Next time consider just deleting it instead.